### PR TITLE
Change opencl package to reduce rate of asum exceptional failures. 

### PR DESCRIPTION
**Pull request by AdamHarries**
This PR changes the intel opencl package to a more up to date one, in order to avoid failures in the asum test. Additionally, the verbosity of the `make` command is reduced in order to make following the output of the build process easier.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ function configure {
 }
 
 function mak {
-    pushd build && make VERBOSE=1 -j$NPROC
+    pushd build && make -j$NPROC
```
Comment (RossBrunton) [inline build.sh]: Can pushd fail? If so, do you want to popd?

Comment (Nyrio) [inline build.sh]: Why this change?

Comment (mehdi-goli) [reply build.sh]: @Nyrio this is because the length of the log would be more that 10000 lines in travis and then travis cannot shows the logs and any result should be seen in a raw log. That is why we change that to be able to see the travis result.

Comment (mehdi-goli) [reply build.sh]: @RossBrunton  this is because the length of the log would be more that 10000 lines in travis and then travis cannot shows the logs and any result should be seen in a raw log. That is why we change that to be able to see the travis result.

Comment (RossBrunton) [reply build.sh]: We have safeguards in place for when pushd fails... But if it does, then nothing will be on the stack when popd runs.

(This is nitpicky, and I don't see it as a blocker)

Comment (RossBrunton) [reply build.sh]: But that's a different issue, so shouldn't be fixed here.
